packages/block-library/src/media-text/edit.js

```diff
@@ -73,6 +108,10 @@ class MediaTextEdit extends Component {
 			src = get( media, [ 'sizes', 'large', 'url' ] ) || get( media, [ 'media_details', 'sizes', 'large', 'source_url' ] );
 		}
 
+		this.setState( {
+			isEditing: false,
```

Comment:
Should this trigger `speak( __( 'You are now viewing the image in the image block.' ) )`?